Android/app/src/main/java/use_case/userList/UserList.java

```diff
@@ -35,26 +37,27 @@ public UserList() {
     public UserList(String filepath) {
         this.filepath = filepath;
         readWriter = new SerReadWriter();
-        UserList.users =readWriter.readFromFile(filepath);
+        users = readWriter.readFromFile(filepath);
     }
 
 
-
-
     /**
      * Add user to this user list.
      *
      * @param user the user to add
      */
-    public static void addUser(User user) {
+    public void addUser(User user) {
         users.put(user.getId(), user);
     }
 
     public String addNewUser(String id, String name, String password) {
```

Comment:
needs javadoc

Reply:
this method is redundant now, probably will be removed